llm/client.py

```
@@ -43,21 +48,40 @@ def _get_mock(self) -> str:
 
-    def structured_chat(self, messages: List[Dict[str, str]], response_model: Type[T], temperature: Optional[float] = None) -> T:
+    def structured_chat(
+        self,
+        messages: List[Dict[str, str]],
+        response_model: Type[T],
+        temperature: Optional[float] = None,
+    ) -> T:
         if self.mock_enabled:
             try:
                 return response_model.model_validate(json.loads(self._get_mock()))
             except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)